src/test/java/com/digitalasset/testing/PingPongIT.java

```diff
@@ -72,76 +74,83 @@ public void testCreate() throws InvalidProtocolBufferException {
   @Test
   public void testObservationWithMatcher() throws InvalidProtocolBufferException {
     Record recordMatcher =
-        record(field("sender", ALICE), field("receiver", BOB), field("count", int64(2)));
+        record(field("sender", alice), field("receiver", bob), field("count", int64(2)));
 
-    ledger().getCreatedContractId(BOB, pingTemplateId(), recordMatcher, ContractId::new);
+    ledger().getCreatedContractId(bob, pingTemplateId(), recordMatcher, ContractId::new);
   }
 
   @Test
   public void testObservationWithoutMatcher() throws InvalidProtocolBufferException {
-    ledger().getCreatedContractId(BOB, pingTemplateId(), ContractId::new);
+    Party bob = getUniqueParty("Bob");
```

Comment:
why is this necessary? can you remove?

Reply:
Yes, this should not be here, I should have removed all these

Reply:
Done.